CMakeLists.txt

```diff
@@ -132,7 +132,7 @@ IF(WANT_QT5)
 
 	FIND_PACKAGE(Qt5Core REQUIRED)
 	FIND_PACKAGE(Qt5Gui REQUIRED)
-	FIND_PACKAGE(Qt5LinguistTools REQUIRED)
+	FIND_PACKAGE(Qt5LinguistTools)
```

Comment:
`lupdate` doesn't exist for Windows QT4 builds, so making this dependency mandatory for Qt5 doesn't make much sense.

Tested this by echoing `${QT_LUPDATE_EXECUTABLE}` in a Windows qt4 build and it's not found, nor does `mingw` have a version available.